ui/utils/index.js

```diff
@@ -0,0 +1,10 @@
+export const debounce = (callback, wait = 200) => {
```

Comment:
is this used?

Reply:
Yes, I am using it for the report schedule search input field in the Create Feed Dialog (not in this PR). I added it now so I can forget about it in my other branch.